### PR TITLE
Individual course enrollment view for manager

### PR DESCRIPTION
**Pull request by mehalshah**
image

### CODE REVIEW

**Comment by mehalshah** [comment]
Right now, I'm just opening this to admins. We'll have to do some logic to open it to PLP and District Managers in the near future

File: dashboard/app/models/ability.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@ def initialize(user)
         can :manage, UserLevel do |user_level|
           !user.students.where(id: user_level.user_id).empty?
         end
-        can :read, Plc::UserCourseEnrollment
+        can :index, Plc::UserCourseEnrollment
```
Comment (aoby) [inline dashboard/app/models/ability.rb]: Why change it to index? That's just an alias for read. See https://github.com/CanCanCommunity/cancancan/wiki/Action-Aliases

Unless you're specifically checking for that custom action (though I'm not sure how that will work with the alias).

**Comment by aoby** [comment]
lgtm, w minor comment

File: dashboard/app/models/ability.rb (hunk 1 above)
Comment (mehalshah) [inline dashboard/app/models/ability.rb]: I'm not - I'll change it back

**Pull request merged**